lib/cpp/CMakeLists.txt

```diff
@@ -103,14 +103,7 @@ if(OPENSSL_FOUND AND WITH_OPENSSL)
 endif()
 
 # WITH_*THREADS selects which threading library to use
-if(WITH_BOOSTTHREADS)
-    set( thriftcpp_threads_SOURCES
-        src/thrift/concurrency/BoostThreadFactory.cpp
-        src/thrift/concurrency/BoostMonitor.cpp
-        src/thrift/concurrency/BoostMutex.cpp
-    )
-    list(APPEND SYSLIBS "${Boost_LIBRARIES}")
-elseif(UNIX AND NOT WITH_STDTHREADS)
+if(UNIX AND NOT WITH_STDTHREADS)
```

Comment:
We don't need pthread raw support any more.  Only use std::threads, get rid of all the options?  Perhaps in the next round?

Reply:
yes，next round